react/src/router/config.tsx

```diff
@@ -1,5 +1,27 @@
 import { App } from '@/App';
-import { ErrorLayout, AuthLayout, MainLayout } from '@/layouts';
+import { ErrorLayout, AuthLayout, MainLayout, EmptyLayout } from '@/layouts';
+import SignIn from '@/pages/sign-in/SignIn';
+import SignUp from '@/pages/sign-up/SignUp';
+import PasswordRecover from '@/pages/password-recovery/PasswordRecover';
+import SmsConfirmation from '@/pages/sms-confirmation/SmsConfirmation';
+import ConfirmEmail from '@/pages/confirm-email/ConfirmEmail';
+import Home from '@/pages/home/Home';
+import Profile from '@/pages/profile/Profile';
+import Friends from '@/pages/friends/Friends';
+import Messages from '@/pages/messages/Messages';
+import Music from '@/pages/music/Music';
+import News from '@/pages/news/News';
+import Posts from '@/pages/posts/Posts';
+import Schedule from '@/pages/schedule/Schedule';
+import Settings from '@/pages/settings/Settings';
+import Shop from '@/pages/shop/Shop';
+import Weather from '@/pages/weather/Weather';
+import Video from '@/pages/video/Video';
+import Map from '@/pages/map/Map';
+import StockExchange from '@/pages/stock-exchange/StockExchange';
+import Survey from '@/pages/survey/Survey';
+import Users from '@/pages/users/Users';
+import User from '@/pages/user/User';
```

Comment:
we can make paths even shorter using index.ts files and capital letters in folder naming, e.g. 
`...from '@/pages/User'`

Reply:
same reason